deployment/kafkas3backup-stream-config.json.template

```diff
@@ -0,0 +1,34 @@
+{
+  "container": {
+    "type": "DOCKER",
+    "docker": {
+      "image": "mastodonc/tp-streams:git-55c906e3b35d",
+      "parameters": [
+         {"key": "log-driver", "value": "gelf"},
+         {"key": "log-opt", "value": "gelf-address=udp://logstash.vpc-vpc.kixi:12201"},
+         {"key": "label", "value": "cluster=momondo"},
+         {"key": "label", "value": "application=travelport.streams"},
+	 {"key": "shm-size", "value": "1536m"},
+	 {"key": "env", "value": "ZK_CONNECT=master.mesos:2181/dcos-service-kafka"},
+         {"key": "env", "value": "AWS_ACCESS_KEY_ID=@@AWS_ACCESS_KEY_ID@@"},
+         {"key": "env", "value": "AWS_SECRET_KEY=@@AWS_SECRET_KEY@@"},
+         {"key": "env", "value": "KAFKA_TOPIC=@@KAFKA_TOPIC@@"},
+         {"key": "env", "value": "KAFKA_CONSUMER_GROUP=@@KAFKA_CONSUMER_GROUP@@"},
+         {"key": "env", "value": "S3_BUCKET=@@S3_BUCKET@@"},
+         {"key": "env", "value": "NUMBER=@@NUMBER@@"}
+      ],
+      "network": "BRIDGE",
+      "portMappings": [
+          {
+              "containerPort": 18082,
+              "hostPort": 18082,
+              "protocol": "tcp"
+          }
+      ]
+    }
+  },
+  "mem": 2048,
+  "cpus": 1,
+  "instances": 1,
+  "id": "tp-streams@@NUMBER@@"
```

Comment:
Just went with how we've done things in the past. For ease I'd prefer to go with what we know.